custom/icds_reports/tests/test_citus_comparison.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import absolute_import, unicode_literals
+
+from datetime import datetime
+
+from django.test import TestCase
+
+from mock import patch
+
+from corehq.util.test_utils import flag_disabled, flag_enabled
+from custom.icds_reports.models import AggAwc
+from custom.icds_reports.utils import get_location_level
+from custom.icds_reports.views import FactSheetsReport
+
+
+class TestDjango(TestCase):
+    @flag_disabled('ICDS_COMPARE_QUERIES_AGAINST_CITUS')
+    @patch('custom.icds_reports.utils.tasks.call_citus_experiment')
+    def test_compare_not_called(self, experiment):
+        list(AggAwc.objects.values('awc_id'))
+        experiment.assert_not_called()
+
+    @flag_enabled('ICDS_COMPARE_QUERIES_AGAINST_CITUS')
+    @patch('custom.icds_reports.utils.tasks.call_citus_experiment')
+    def test_compare_called_no_filter(self, experiment):
+        list(AggAwc.objects.values('awc_id'))
+        self.assertEqual(len(experiment.call_args_list), 1)
+        call = experiment.call_args_list[0]
+        self.assertEqual(call[0][0], 'SELECT "agg_awc"."awc_id" FROM "agg_awc"')
+        self.assertEqual(call[0][1], [])
+        self.assertEqual(call[1]['data_source'], 'AggAwc')
+
+    @flag_enabled('ICDS_COMPARE_QUERIES_AGAINST_CITUS')
+    @patch('custom.icds_reports.utils.tasks.call_citus_experiment')
+    def test_compare_called_with_filter(self, experiment):
```

Comment:
@emord This test is broken in Python 3 - could you please take a look?

https://api.travis-ci.org/v3/job/532417095/log.txt